generators/entity/templates/server/src/main/resources/config/liquibase/changelog/_added_entity.xml

```diff
@@ -55,6 +55,10 @@
             if (fields[idx].fieldValidate === true && fields[idx].fieldValidateRules.indexOf('required') !== -1) {
                 nullable = false;
             }
+            let unique = false
+            if (fields[idx].fieldValidate === true && fields[idx].fieldValidateRules.indexOf('unique') !== -1) {
```

Comment:
use `fields[idx].fieldValidateRules.includes('unique')` instaed of `fields[idx].fieldValidateRules.indexOf('unique') !== -1` I'm changing other usages